src/js/Controllers/TTSController.js

```diff
@@ -123,13 +131,21 @@ class TTSController {
                 // in case more things will be added to the filePlaylist,
                 // so that TTS Speak will be replied to before speaking next message
                 clearInterval(this.timers[file.id]);
+                this.speakEnded();
```

Comment:
```suggestion
```

There may be more to speak after REPLY is sent, for ex is GUFM response arrives during TTS.Speak

Reply:
@iCollin please find fix [here](https://github.com/LuxoftSDL/generic_hmi/pull/35/files)